docs/README.md

```diff
@@ -178,58 +182,51 @@ When invoked with the `--offline` parameter, the CLI performs a preboot sequence
-    restart [--pull | --force]           Restart ${CHE_MINI_PRODUCT_NAME} services
-    destroy                              Stops services, and deletes ${CHE_MINI_PRODUCT_NAME} instance data
-            [--quiet                         Does not ask for confirmation before destroying instance data
-             --cli]                          If :/cli is mounted, will destroy the cli.log
-    rmi [--quiet]                        Removes the Docker images for <version>, forcing a repull
-    config                               Generates a ${CHE_MINI_PRODUCT_NAME} config from vars; run on any start / restart
-    add-node                             Adds a physical node to serve workspaces intto the ${CHE_MINI_PRODUCT_NAME} cluster
-    remove-node <ip>                     Removes the physical node from the ${CHE_MINI_PRODUCT_NAME} cluster
-    upgrade                              Upgrades Che from one version to another with migrations and backups
-    download [--pull|--force|--offline]  Pulls Docker images for the current che-cli version
-    backup [--quiet | --skip-data]       Backups ${CHE_MINI_PRODUCT_NAME} configuration and data to /che/backup volume mount
-    restore [--quiet]                    Restores ${CHE_MINI_PRODUCT_NAME} configuration and data from /che/backup mount
-    offline                              Saves ${CHE_MINI_PRODUCT_NAME} Docker images into TAR files for offline install
-    info                                 Displays info about ${CHE_MINI_PRODUCT_NAME} and the CLI 
-         [ --all                             Run all debugging tests
-           --debug                           Displays system information
-           --network]                        Test connectivity between ${CHE_MINI_PRODUCT_NAME} sub-systems
-    ssh <wksp-name> [machine-name]       SSH to a workspace if SSH agent enabled
-    mount <wksp-name>                    Synchronize workspace with current working directory
-    action <action-name> [--help]        Start action on ${CHE_MINI_PRODUCT_NAME} instance
-    compile <mvn-command>                SDK - Builds Che source code or modules
-    test <test-name> [--help]            Start test on ${CHE_MINI_PRODUCT_NAME} instance
-
-Variables:
-    CHE_HOST                             IP address or hostname where ${CHE_MINI_PRODUCT_NAME} will serve its users
-    CLI_DEBUG                            Default=false.Prints stack trace during execution
-    CLI_INFO                             Default=true. Prints out INFO messages to standard out
-    CLI_WARN                             Default=true. Prints WARN messages to standard out
-    CLI_LOG                              Default=true. Prints messages to cli.log file
+USAGE: 
+  docker run <DOCKER_PARAMETERS> ${CHE_IMAGE_FULLNAME} [COMMAND]
+
+MANDATORY DOCKER PARAMETERS:
+  -v <LOCAL_PATH>:${CHE_CONTAINER_ROOT}         Where user, instance, and log data saved
```

Comment:
I'm not sure all the variables ${CHE_CONTAINER_ROOT}, ${CHE_FORMAL_PRODUCT_NAME} will work there ?